packages/paste-website/src/components/tokens-list/utils/google-analytics.ts

```diff
@@ -0,0 +1,15 @@
+// import {trackCustomEvent} from 'gatsby-plugin-google-analytics';
+import debounce from 'lodash/debounce';
+
+export const trackTokenFilterString = debounce((filter: string): void => {
+  if (filter !== '') {
+    console.info('[DEV]: track function called.');
```

Comment:
ℹ️ Added this because I didn't want to trigger a bunch of these in dev and it isn't clear to me if these are mocked or blocked from submitting data in dev. @zahnster Any thoughts/context here?

Reply:
Should be safe in dev. Source: https://www.gatsbyjs.com/plugins/gatsby-plugin-google-analytics/

> Note that this plugin is disabled while running gatsby develop. This way, actions are not tracked while you are still developing your project. Once you run gatsby build the plugin is enabled. Test it with gatsby serve.

Reply:
Awesome, good to know. Thanks!